### DISCUSSION TITLE
Request support for CDN, which means setting up parent node resolution

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/interactsh#729] [Ideas]
@ehsandeep 
Request support for CDN, which means setting up parent node resolution

Usage Scenario

In order to protect server A, a front-end CDN has been set. At this time, the DNS IP is dynamic, so this IP needs to be set with a front-end resolution server, such as

aiden.ns.cloudflare.com

journey.ns.cloudflare.com

Thus returning the dynamic IP address

thanks

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
